### PR TITLE
Feature add readme

### PR DESCRIPTION
**Pull request by ko4edikov**

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ logging:
   level:
```
Comment (savand) [inline src/main/resources/application.yml:3]: no need to change this file

File: pom.xml [hunk 2]
```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>org.apache.tomcat.embed</groupId>
       <artifactId>tomcat-embed-jasper</artifactId>
-      <scope>provided</scope>
```
Comment (savand) [inline pom.xml:47]: this change is local for idea, fine working with provided scope, don't change on master

File: familyHealthKeeper.iml [hunk 3]
```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (savand) [inline familyHealthKeeper.iml:1]: what are the reasons to add this file to the master

**Review by savand — COMMENTED** [review]
Look through the comments below

File: familyHealthKeeper.iml (hunk 3 above)
Comment (ko4edikov) [reply familyHealthKeeper.iml:1]: Ok. I'll fix it.

**Pull request closed without merging**